util/test/sub_test

```diff
@@ -787,14 +787,12 @@ else:
 #   or not this is a good idea, but preserving it for now for backwards
 #   compatibility.
 #
+
+directoryCompopts = list(' ')
```

Comment:
Yes, it's because I wanted the for dir_compopts .... loop computing the product to return something.
directoryCompopts is not used except in building the product (which ends up in compoptslist),
so this should be OK.